Security/Authentication/Token/PreAuthenticationJWTUserToken.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Lexik\Bundle\JWTAuthenticationBundle\Security\Authentication\Token;
+
+use Symfony\Component\Security\Guard\Token\PreAuthenticationGuardToken;
+
+/**
+ * PreAuthenticationJWTUserToken.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class PreAuthenticationJWTUserToken extends PreAuthenticationGuardToken
```

Comment:
I like that you've created a nice object that can be returned from getCredentials(), instead of an array. But, I don't think there's any advantage to extending `PreAuthenticationGuardToken` - you can really just have a class that doesn't return *anything*.